webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/__tests__/integration.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { IntegrationTestHelper } from '@theforeman/test';
+import { Router } from 'react-router-dom';
+
+import ConnectedWebhooksIndexPage from '../index';
+
+import * as selectors from '../../WebhooksPageSelectors';
+import { spySelector } from './WebhooksIndexPage.fixtures';
+
+spySelector(selectors);
+
+describe('WebhooksIndexPage - Integration Test', () => {
+  it('should flow', () => {
+    const integrationTestHelper = new IntegrationTestHelper();
+
+    const component = integrationTestHelper.mount(<Router history={{}}>
```

Comment:
and then here you can use it in `history={createMemoryHistory()}`

Reply:
otherwise, you can pass here an object which contains all functionality you need, e.g:
```js
{
 listen: jest.fn,
 action: 'POP',
 etc....
}
```